### PR TITLE
Implement the full new workflow from question creation to answer viewing

### PR DESCRIPTION
**Pull request by maximusunc**

### CODE REVIEW

File: frontend/src/utils/queryGraph.js [hunk 1]
```diff
@@ -232,12 +244,27 @@ function prune(q_graph) {
   Object.keys(clonedQueryGraph.nodes).forEach((n) => {
     pruneEmptyArray(clonedQueryGraph.nodes[n], 'category');
     pruneEmptyArray(clonedQueryGraph.nodes[n], 'id');
-    if (clonedQueryGraph.nodes[n].id && clonedQueryGraph.nodes[n].category) {
+    // if (clonedQueryGraph.nodes[n].id && clonedQueryGraph.nodes[n].category) {
+    //   delete clonedQueryGraph.nodes[n].category;
```
Comment (sarah-seitanakis) [inline frontend/src/utils/queryGraph.js:248]: Can these commented lines be removed?

File: frontend/tests/common/test_message.json [hunk 2]
```diff
@@ -41,7 +41,7 @@
       {
         "node_bindings": {
           "n1": [{ "id": "1" }],
-          "n2": [{ "id": "2" }]
+          "n0": [{ "id": "2" }]
```
Comment (sarah-seitanakis) [inline frontend/tests/common/test_message.json:44]: I know that this is just semantic and doesn't really affect anything, but wouldn't it make more sense if `n0` came before `n1` with the `id` of `n0` as 1 and the `id` of `n1` as 2? Just a thought.

File: query-dispatcher/routes.js [hunk 3]
```diff
@@ -6,8 +6,9 @@ const { handleAxiosError } = require('./utils');
 
 // const messenger = process.env.MESSENGER_URL ||
 //                    'http://robokop.renci.org:4868/answer';
-const strider = process.env.STRIDER_URL ||
-                    'http://robokop.renci.org:5781';
+// const strider = process.env.STRIDER_URL ||
+//                     'http://robokop.renci.org:5781';
+const robokop = process.env.ROBOKOP_URL || 'http://robokop.renci.org:7092';
```
Comment (sarah-seitanakis) [inline query-dispatcher/routes.js:11]: Is this referring to the external service RoboKop or is this for qgraph?

Comment (maximusunc) [reply query-dispatcher/routes.js:11]: This is the external Robokop ARA.

**Review by sarah-seitanakis — CHANGES_REQUESTED** [review]
You might want to go through and make sure that variables referring to the UI that contain `robokop` are changed to `qgraph`. I think it might be easier to catch these sooner, rather than later.

**Comment by maximusunc** [comment]
> You might want to go through and make sure that variables referring to the UI that contain `robokop` are changed to `qgraph`. I think it might be easier to catch these sooner, rather than later.

Agreed. There will be a rebranding PR that will update all of our variables and text to reference 'qgraph'.

File: frontend/tests/common/test_message.json (hunk 2 above)
Comment (maximusunc) [reply frontend/tests/common/test_message.json]: Updated in 6354b586c0bbfa1213a8289bbdcce81e1e1816d9

File: frontend/src/utils/queryGraph.js (hunk 1 above)
Comment (maximusunc) [reply frontend/src/utils/queryGraph.js]: Removed in 74df36f5ada1baa2a9771ec61dfc50831e1aa063

**Review by sarah-seitanakis — APPROVED** [review]

File: frontend/tests/common/test_message.json [hunk 4]
```diff
@@ -2,13 +2,13 @@
...
-          "id": "MONDO:0005737",
-          "is_set": false
-        },
         "n0": {
-          "category": "biolink:Gene",
+          "categories": ["biolink:Gene"],
           "is_set": true
+        },
+        "n1": {
+          "id": ["MONDO:0005737"],
```
Comment (patrickkwang) [inline frontend/tests/common/test_message.json:10]: In TRAPI 1.1, this should be "ids".

**Review by patrickkwang — APPROVED** [review]
Looks good! I'll add some issues for future development.

File: frontend/tests/common/test_message.json (hunk 4 above)
Comment (maximusunc) [reply frontend/tests/common/test_message.json:10]: We talked offline. The UI is currently between TRAPI 1.0 and 1.1, and this is a side effect. There will be a future PR that will transition us completely to 1.1.

**Pull request merged**